test/pkg/environment/aws/expectations.go

```diff
@@ -143,70 +145,73 @@ func (env *Environment) GetInstanceProfileName(nodeClass *v1.EC2NodeClass) strin
 	return fmt.Sprintf("%s_%d", env.ClusterName, lo.Must(hashstructure.Hash(fmt.Sprintf("%s%s", env.Region, nodeClass.Name), hashstructure.FormatV2, nil)))
 }
 
-func (env *Environment) GetInstance(nodeName string) ec2.Instance {
+func (env *Environment) GetInstance(nodeName string) ec2types.Instance {
 	node := env.Environment.GetNode(nodeName)
 	return env.GetInstanceByID(env.ExpectParsedProviderID(node.Spec.ProviderID))
 }
 
 func (env *Environment) ExpectInstanceStopped(nodeName string) {
 	GinkgoHelper()
 	node := env.Environment.GetNode(nodeName)
-	_, err := env.EC2API.StopInstances(&ec2.StopInstancesInput{
+	_, err := env.EC2API.StopInstances(context.Background(), &ec2.StopInstancesInput{
 		Force:       aws.Bool(true),
-		InstanceIds: aws.StringSlice([]string{env.ExpectParsedProviderID(node.Spec.ProviderID)}),
+		InstanceIds: []string{env.ExpectParsedProviderID(node.Spec.ProviderID)},
 	})
 	Expect(err).To(Succeed())
 }
 
 func (env *Environment) ExpectInstanceTerminated(nodeName string) {
 	GinkgoHelper()
 	node := env.Environment.GetNode(nodeName)
-	_, err := env.EC2API.TerminateInstances(&ec2.TerminateInstancesInput{
-		InstanceIds: aws.StringSlice([]string{env.ExpectParsedProviderID(node.Spec.ProviderID)}),
+	_, err := env.EC2API.TerminateInstances(context.Background(), &ec2.TerminateInstancesInput{
```

Comment:
We should use the evn context
```suggestion
	_, err := env.EC2API.TerminateInstances(env.Context, &ec2.TerminateInstancesInput{
```